cmd/ext/create.go

```diff
@@ -298,9 +299,13 @@ func invokeCreate(args []string) {
 			Doing: "Creating", Done: "created", On: "'localdev' client extension project", Enable: !flags.Verbose,
 		},
 		func(fior func(io.ReadCloser, bool, string) int) int {
-			return docker.InvokeCommandInLocaldev("localdev-ext-create", config, host, true, flags.Verbose, fior, "")
+			return docker.InvokeCommandInLocaldev("localdev-ext-ref", config, host, true, flags.Verbose, fior, "")
```

Comment:
Wondering why we changed the name of container to `localdev-ext-ref`? is that for `refresh`?

Reply:
good catch, thx. That's called changing your mind mid-edit... eliminated that change.